recipes/imbalanced-learn/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "imbalanced-learn" %}
+{% set version = "0.2.0.dev0" %}
+package:
+  name: imbalanced-learn
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ed97184e5665ea71b44222541b4c3d36daf4a8f6cf402df2976538082ff0509d
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.10.4
+    - scipy >=0.17.0
+    - scikit-learn >=0.17.1
+  run:
+    - python
+    - numpy >=1.10.4
+    - scipy >=0.17.0
+    - scikit-learn >=0.17.1
+
+test:
+  requires:
+    - nose
+    - coverage
+  imports:
+    - imblearn
+
+about:
+  home: https://github.com/scikit-learn-contrib/{{ name }}
+  license: MIT
+  license_file: LICENSE
+  summary: 'Python module to perform under sampling and over sampling with various techniques'
+  description: |
+    imbalanced-learn is a python package offering a number of re-sampling techniques commonly
+    used in datasets showing strong between-class imbalance. It is compatible with scikit-learn
+    and is part of scikit-learn-contrib projects.
+  doc_url: http://contrib.scikit-learn.org/{{ name }}
+  dev_url: https://github.com/scikit-learn-contrib/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - Guillaume Lemaitre
```

Comment:
Please change to your GitHub handle, @glemaitre . This is needed to give you permissions on the feedstock once it is created.